.travis.yml

```diff
@@ -20,17 +20,14 @@ matrix:
     - php: 5.6
       env: SYMFONY_VERSION=2.6.*
```

Comment:
PHP 5.4 and 5.5 *are* tested. They are running on builds which are not modifying the Symfony requirement